content/webapp/views/components/ImageWithTasl/ImageWithTasl.WorkLink.tsx

```diff
@@ -3,21 +3,21 @@ import styled from 'styled-components';
 import { font } from '@weco/common/utils/classnames';
 import Space from '@weco/common/views/components/styled/Space';
 
-const WorkLinkContainer = styled(Space).attrs({
+export const WorkLinkContainer = styled(Space).attrs({
```

Comment:
This is why I think maybe it needs to be taken out and be its own component?

Reply:
in the styled directory do you think?

Reply:
I see it as this subcomponent being removed, becoming `FeaturedWorkLink` as its own component in `views/components/content`. It would take in a variant such as `variant="inline" | "image"` , which would returns the relevant component. We've done it in a few components already, https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/views/components/InfoBanner/index.tsx would be an example.
It's just the exporting of a sub-component's style, I feel like it could benefit from centralisation, which then can become a good cardigan story